.github/scripts/GitHub.py

```diff
@@ -221,16 +221,22 @@ def add_reviewers_to_pr(
             f"::error title=User is not a Collaborator!::{', '.join(non_collaborators)}"

```

Comment:
Comment on commit message:
"If a non-collaborator is part of the reviewer list, an admin needs
to be notified so they can be removed."

Unsure of meaning. Maybe tainted by the current state, that we have a bunch of non-collaborators that we need to *add*.

The change itself is very welcome.

Reply:
Apart from the above question, I'm happy with this PR - many thanks for the rework, it made for a lot easier review.

Reply:
The new change isolates the list of non-collaborators, this makes it clear exactly who needs to be added. In response to this comment, I also added admin instructions to clarify next steps for those users. Here's an example of a non-collaborator notification with the change just pushed.

<img width="749" alt="image" src="https://github.com/user-attachments/assets/e8f4f2cc-f58a-46d3-8545-6dc80ea0f5ae">

Reply:
Right, but that's the output (which is great). My comment was on the commit message.

Reply:
I see. There are also some with emails that bounce back and are inactive, so I drifted down the case of removal when writing that. The new comment revision should clarify this in practice. Thanks for the feedback.